pkg/agent/agent_windows.go

```diff
@@ -307,3 +448,56 @@ func (i *Initializer) setInterfaceMTU(iface string, mtu int) error {
 	}
 	return util.SetInterfaceMTU(iface, mtu)
 }
+
+func (i *Initializer) setVMNodeConfig(en *v1alpha1.ExternalNode, nodeName string) error {
+	// TODO: Handle for multiple interfaces
+	var uplinkInterface *net.Interface
+	foundNetDevice := false
+	for _, addr := range en.Spec.Interfaces[0].IPs {
+		var ipFilter *utilip.DualStackIPs
+		var err error
+		epIP := net.ParseIP(addr)
+		if epIP.To4() != nil {
+			ipFilter = &utilip.DualStackIPs{IPv4: epIP}
+		} else {
+			ipFilter = &utilip.DualStackIPs{IPv6: epIP}
+		}
+		_, _, uplinkInterface, err = util.GetIPNetDeviceFromIP(ipFilter, sets.NewString())
+		if err != nil {
+			klog.InfoS("Unable to get net device by IP", "IP", addr)
+		} else {
+			foundNetDevice = true
+			klog.V(2).InfoS("Net device found on the ExternalNode", "interfaceName", uplinkInterface.Name)
+			break
+		}
+	}
+	if !foundNetDevice {
+		return fmt.Errorf("failed to get net device for externalnode %s", en.Name)
+	}
+	i.nodeConfig = &config.NodeConfig{
+		Name:            nodeName,
+		Type:            config.ExternalNode,
+		OVSBridge:       i.ovsBridge,
+		UplinkNetConfig: new(config.AdapterNetConfig),
+	}
+	i.nodeConfig.UplinkNetConfig.Name = uplinkInterface.Name
+	return nil
+}
+
+// installVMFlows configures default flows between uplink and host port,
+// so that antrea-agent can connect to antrea-controller.
+func (i *Initializer) installVMInitialFlows() error {
+	hostIfConfig, found := i.ifaceStore.GetInterfaceByName(i.nodeConfig.UplinkNetConfig.Name)
+	if found {
```

Comment:
handle not found (error) case first, then major code (normal case) don't need to indent

Reply:
Done